### PR TITLE
feat (Python binding): Make API calls up to date

### PR DESCRIPTION
**Pull request by bingyanglin**
- Update API calls
- Add a simple python binding example

### CODE REVIEW

File: bindings/python/native/iota_client/client.py [hunk 1]
```diff
@@ -84,3 +155,34 @@ def unsynced_nodes(self):
...
+    def store_mnemonic(self, secret_manager, mnemonic):
+        """Store a mnemonic in the Stronghold vault.
+        """
+        return self.call_client_method('StoreMnemonic', {
+            'secret_manager': secret_manager,
+            'mnemonic': mnemonic
+        })
+
+    def submit_payload(self, payload_dto):
+        """Store a mnemonic in the Stronghold vault.
```
Comment (Thoralf-M) [inline bindings/python/native/iota_client/client.py]: Wrong description

Comment (bingyanglin) [reply bindings/python/native/iota_client/client.py]: Thanks for pointing this out. Modified at [45d47b3](https://github.com/iotaledger/iota.rs/pull/1055/commits/45d47b33738532b58b0b0f52951dd794dc8954f9)

**Review by Thoralf-M — DISMISSED** [review]

File: bindings/python/native/iota_client/client.py [hunk 2]
```diff
@@ -19,19 +19,95 @@ def __init__(self, client_config=None):
...
+                           amount=None,
+                           native_tokens=None,
+                           state_index=None,
+                           state_metadata=None,
+                           foundry_counter=None,
+                           features=None,
+                           immutable_features=None):
+        """Build an AliasOutput.
+        """
+        return self.call_client_method('GenerateAddresses', {
```
Comment (thibault-martinez) [inline bindings/python/native/iota_client/client.py]: That seems incorrect

**Review by thibault-martinez — CHANGES_REQUESTED** [review]

File: bindings/python/native/iota_client/client.py (hunk 2 above)
Comment (bingyanglin) [reply bindings/python/native/iota_client/client.py]: Modified. Thanks for pointing this out!

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**